core/templates/dev/head/base.html

```diff
@@ -220,9 +216,58 @@
                             </ul>
                           </li>
                         {% endif %}
-                      </ul>
+                    {% if nav_mode != 'create' and nav_mode != 'explore' %}
```

Comment:
Indentation off. This should be indented by 4 spaces to match the previous {% endif %}.